cfdilib/cfdv32.py

```diff
@@ -157,6 +157,39 @@ def set_xslt(self):
         #       we can not use different algorithms here
         self.xstl = super(Payroll12, self).set_xslt()
 
+
 def get_payroll(dict_payroll, debug_mode=False):
     return Payroll12(dict_payroll, debug_mode=debug_mode)
 
+
+class CFDv33(BaseDocument):
```

Comment:
You are edditing cfdilib/cfdv32.py this should be in cfdilib/cfdv33.py (new file)